### DISCUSSION TITLE
Cannot import custom F# library to Python

### DISCUSSION THREAD

**Discussion by lHeidbreder** [pythonnet/pythonnet#1778] [Q&A]
### Environment

-   Pythonnet version: 3.0.0a2
-   Python version: 3.10
-   Operating System: Win10
-   .NET Runtime: 6.0

### Details

I created an F# library and was trying to use a method in python. I'm running into trouble even importing my module.

F# library consists of one file, looking like this:
```namespace FSTestLib

module TestModule =
    let test =
        12
```

And the failing Python code looks like this:
```import sys
sys.path.append(r"C:\Users\David\source\repos\FSTestLib\FSTestLib\bin\Debug\net6.0")
import clr
clr.AddReference("FSTestLib")

import FSTestLib
#import TestModule
```

I have tried uncommenting either one of the bottom two lines, always failing with a similar traceback, as seen below. The module name obviously depends on the line of code that is active when executing.
```Traceback (most recent call last):
  File "<executing directory>\pynet_test.py", line 6, in <module>
    import FSTestLib
ModuleNotFoundError: No module named 'FSTestLib'
```

**Comment by richClubb** [comment]
Having a similar issue.

**Comment by TonyCongqianWang** [comment]
I had a similar issue with a C# assembly too. Have you tried adding `<Platforms>x64</Platforms>` to your csproj file or whatever is the equivalent in F#. That solved it for me
